el/index.js

```diff
@@ -38,6 +38,13 @@ module.exports = yeoman.generators.Base.extend({
         'ex: yo polymer:el my-element'
       ));
     }
+    
+    var isInsideProjectFolder = this.fs.exists('app/elements/elements.html');
+    if (!isInsideProjectFolder) {
+      this.emit('error', new Error(
+        'Element must be created inside a project folder'
```

Comment:
Hm, this may not be clear for folks. I think we'll want to add a longer explanation to the readme and link to it from this error as well.